Workbooks/Azure Stack HCI/AtScale/Insights.workbook

```diff
@@ -0,0 +1,2056 @@
+                    "size": 3,
+                    "showAnalytics": true,
+                    "timeContext": {
+                      "durationMs": 604800000
+                    },
+                    "timeContextFromParameter": "timeRange",
+                    "showRefreshButton": true,
+                    "queryType": 0,
+                    "resourceType": "microsoft.resources/subscriptions",
+                    "crossComponentResources": [
+                      "{subscription}"
+                    ],
+                    "visualization": "tiles",
+                    "tileSettings": {
+                      "titleContent": {
+                        "columnMatch": "Column",
+                        "formatter": 1
+                      },
+                      "leftContent": {
+                        "columnMatch": "Value",
+                        "formatter": 12,
+                        "formatOptions": {
+                          "palette": "none"
+                        }
+                      },
+                      "rightContent": {
+                        "columnMatch": "Column",
+                        "formatter": 18,
+                        "formatOptions": {
+                          "thresholdsOptions": "icons",
+                          "thresholdsGrid": [
+                            {
+                              "operator": "==",
+                              "thresholdValue": "Healthy",
+                              "representation": "success",
+                              "text": ""
+                            },
+                            {
+                              "operator": "==",
+                              "thresholdValue": "Critical",
+                              "representation": "4",
+                              "text": ""
+                            },
+                            {
+                              "operator": "==",
+                              "thresholdValue": "Warning",
+                              "representation": "2",
+                              "text": ""
+                            },
+                            {
+                              "operator": "==",
+                              "thresholdValue": "Other",
+                              "representation": "unknown",
+                              "text": ""
+                            },
+                            {
+                              "operator": "Default",
+                              "thresholdValue": null,
+                              "representation": "Blank",
+                              "text": ""
+                            }
+                          ]
+                        }
+                      },
+                      "showBorder": false,
+                      "size": "auto"
+                    }
+                  },
+                  "name": "query - 1"
```

Comment:
a lot of these names have `query - 1`, `text - 1`, etc. You can consider giving these groups and steps meaningful names

Reply:
I renamed all of them.